### PR TITLE
2.1

### PR DESCRIPTION
**Pull request by henrikbjorn**
Update the formtypes with the latest changes. We should properly change the old method calls for options aswell.

### CODE REVIEW

File: Form/Type/ImageType.php [hunk 1]
```diff
@@ -60,14 +60,14 @@ public function buildView(FormViewInterface $view, FormInterface $form, array $o
             if ('/' !== DIRECTORY_SEPARATOR) {
                 $uri = str_replace(DIRECTORY_SEPARATOR, '/', $uri);
             }
-            $view->setVar('image_uri', $uri);
-        } else if ($form->hasAttribute ('no_image_placeholder_uri') && $uri = $form->getAttribute ('no_image_placeholder_uri')) {
-            $view->setAttribute('image_uri', $uri);
+            $view->vars['image_uri'] = $uri;
+        } else if ($form->hasAttribute ('no_image_placeholder_uri') && $uri = $form->getAttribute('no_image_placeholder_uri')) {
```
Comment (stof) [inline Form/Type/ImageType.php]: please remove the extra space between the method name and the brace

File: composer.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     }],
     "require": {
         "php": ">=5.3.0",
-        "symfony/symfony" : ">=2.0"
+        "symfony/symfony" : "2.*@beta"
```
Comment (stof) [inline composer.json]: `2.*` is too wide. We cannot ensure the compatibility with 2.2, 2.3 & co yet.

and adding `@beta` here is a bad idea. Travis should be able to use the dev versions when testing IMO so you should better set the minimum-stability to dev.

**Comment by digitalkaoz** [comment]
@beberlei any news on this one?

**Comment by DavidBadura** [comment]
The pull request must be merged manually. I'll be watching this in the next days.

**Comment by DavidBadura** [comment]
Fixed in #40 and #41.

**Pull request closed without merging**